From 497efc020db5d3de4b05623a3392a5c29a3e1535 Mon Sep 17 00:00:00 2001
From: Brad Embree <brad@bestpractical.com>
Date: Wed, 13 Dec 2023 07:29:58 -0800
Subject: [PATCH] Add Initial callback for PreviewInSearch

The PreviewInSearch extension needed a callback to modify various paths
to ticket pages so they worked correctly from the preview on the search
results page.
---
 share/html/Elements/ShowHistoryPage | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/share/html/Elements/ShowHistoryPage b/share/html/Elements/ShowHistoryPage
index f6aa296e87..ab1cf3737e 100644
--- a/share/html/Elements/ShowHistoryPage
+++ b/share/html/Elements/ShowHistoryPage
@@ -56,6 +56,8 @@ $ShowHeaders       => 0
 $PathPrefix        => ''
 </%ARGS>
 <%INIT>
+$m->callback( CallbackName => 'Initial', ARGSRef => \%ARGS, Object => $Object, Transactions => $Transactions, Attachments => $Attachments, AttachmentContent => $AttachmentContent );
+
 my $trans_content = {};
 my $trans_attachments = {};
 
-- 
2.39.3 (Apple Git-145)

